solc_json_parser/parser.py

```diff
@@ -153,6 +153,22 @@ def symbols_to_ids_from_ast_v7(ast: Dict[Any, Any]) -> Dict[str, int]:
     return {k: v[0] for m in syms for k, v in m.items()}
 
 
+def get_increased_version(current_version: str) -> str:
+    # convert current version str to Version object
+    current_version_obj = Version(current_version)
+    versions = solcx.get_installable_solc_versions()
+    # reverse versions list so that it is from small to large
+    versions = sorted(versions)
+    # get the next minimum version
+    try:
+        next_version = next(v for v in versions if v > current_version_obj)
+    except StopIteration:
+        exit(1)
```

Comment:
Inside a library you can raise exception but it's not a good idea to quit the process with `exit`